static/main.js

```diff
@@ -869,7 +869,24 @@ function initCodeMirror() {
                     compressed = '»' + compressed + '»';
                     if (compressed.length <= num.length) {
                         cm.replaceRange(compressed, { line: cur.line, ch: cur.ch - num.length }, { line: cur.line, ch: cur.ch });
-                        // Suggested by Copilot, again. Works.
+                        return;
+                    }
+                }
+                const str = line.match(/`([a-z ]|\\`)+`$/)?.[0]?.slice(1, -1);
```

Comment:
Why are we matching backticks within strings when the strings we're compressing can't contain backticks?

Reply:
smth I forgot to change

```suggestion
                const str = line.match(/`[a-z ]+`$/)?.[0]?.slice(1, -1);
```